README.md

```diff
@@ -52,6 +57,11 @@ For more information about how to get started with the Swift Package Manager, ch
 
 Any kind of contributions would be highly appreciated! To start contributing, read the [Contributing Guideline](https://github.com/mugbug/BricksAndTiles/blob/master/CONTRIBUTING.md) for info on how to report issues, submit ideas and submit pull requests!
 
+
+# Requirements
+ -  OS versions iOS 8.0+ / Mac OS X 10.10+
+ -  Xcode version 7.0+,8.0+,9.0+,10.0+,11.0+
+ -  Swift version 2.2/2.3 , 3.2+ /4.x
```

Comment:
I think those aren't accurate, could you check again?

Reply:
okay on it